Formula/kubetail.rb

```diff
@@ -0,0 +1,24 @@
+class Kubetail < Formula
+  desc "Bash script to tail Kubernetes logs from multiple pods at the same time"
```

Comment:
Shouldn't this have a `uses_from_macos "bash"` then?

Reply:
good point.